assets/behaviors/advancedTasksForOreon.behavior

```diff
@@ -0,0 +1,16 @@
+{
+    selector : [
```

Comment:
This and other JSON files light up like 🎄s in IntelliJ due to formatting - our parsing leniency lets it work anyway but just about everything needs to be in double quotes. Make sure to add `.behavior` and other files to match for JSON under file types and it should help you with the formatting :-)